frontend/packages/kubevirt-plugin/src/selectors/immutable/template/combined.ts

```diff
@@ -86,6 +86,13 @@ export const iGetRelevantTemplates = (
         const versionCMP = compareVersions(splitVersion(aVersion), splitVersion(bVersion)) * -1; // descending
 
         if (versionCMP !== 0) {
```

Comment:
can we move this logic to compareVersions and also add it to the unit test?

Reply:
done :+1: